webhook/src/main/java/com/netflix/conductor/contribs/publisher/TaskNotification.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.contribs.publisher;
+
+import java.util.LinkedHashMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.run.TaskSummary;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ser.FilterProvider;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
+
+@JsonFilter("SecretRemovalFilter")
+public class TaskNotification extends TaskSummary {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TaskStatusPublisher.class);
+
+    public String workflowTaskType;
+    private String domainGroupMoId = "";
+    private String accountMoId = "";
```

Comment:
This seems like an application specific field.  The notification should be a generic class.

Reply:
Thank you Viren for reviewing! Yes, both accountMoId and domainGroupMoId are custom fields. I will work on it and get back.

Reply:
Looking forward to your achievements!

Reply:
#https://github.com/charybr/conductor-community/pull/1

Reply:
thank you Huang! for the comments. i will check if we can build url dynamically.

Reply:
> thank you Huang! for the comments. i will check if we can build url dynamically.

Hi, @charybr ! This is my PR regarding this issue(https://github.com/charybr/conductor-community/pull/1). Could you consider merging?

Reply:
thats great Huang! Thank you. Give me 3-4 days i will check and merge by this weekend.

Reply:
Hi Huang, i have merged your PR. thank you!

Reply:
Hi, how's it going?

Reply:
review is pending